src/Extension/FluentDirectorExtension.php

```diff
@@ -70,9 +72,9 @@ public function updateRules(&$rules)
 
         // @todo - Build SitemapController after GoogleSitemap module is migrated to ss4
         // If Google sitemap module is installed then replace default controller with custom controller
-        //if (class_exists(GoogleSitemapController::class)) {
-        //    $rules['sitemap.xml'] = SitemapController::class;
-        //}
+        if (class_exists(GoogleSitemapController::class)) {
+            $rules['sitemap.xml'] = FluentSitemapController::class;
```

Comment:
If you use a fluent-namespaced controller you need the fluent-namespaced template also.

Just use the default googlesitemapcontroller class from will.r's module.

Reply:
If you use the the default Wilr's default googlesitemapcontroller, then the default GoogleSitemapController_sitemap.ss gets rendered and that one doesn't contain the alternate languages. Or is there a way to route the default controller to the FluentSitemapController.ss?